rdt/transformers/utils.py

```diff
@@ -274,3 +274,37 @@ def learn_rounding_digits(data):
         name,
     )
     return None
+
+
+class WarnDict(dict):
+    """Custom dictionary to raise a deprecation warning."""
+
+    def get(self, key):
+        """Retrun the value for key if key is in the dictionary, else default.
+
+        If the key is `text` raises a `DeprecationWarning` stating that it will be
+        phased out.
+        """
+        if key == 'text':
```

Comment:
should we make a DEPRECATED_SDTYPES tuple and check that. We can just put text in it for now